src/GetStarted/Program.java

```diff
@@ -30,50 +62,59 @@ public static void main( String[] args ) throws ExecutionException, InterruptedE
-            - traversals, including loops
-            - update annd delete vertices and edges
-        */
-        String gremlinQueries[] = new String[] {
-            "g.V().drop()",
-            "g.addV('person').property('id', 'thomas').property('firstName', 'Thomas').property('age', 44)",
-            "g.addV('person').property('id', 'mary').property('firstName', 'Mary').property('lastName', 'Andersen').property('age', 39)",
-            "g.addV('person').property('id', 'ben').property('firstName', 'Ben').property('lastName', 'Miller')",
-            "g.addV('person').property('id', 'robin').property('firstName', 'Robin').property('lastName', 'Wakefield')",
-            "g.V('thomas').addE('knows').to(g.V('mary'))",
-            "g.V('thomas').addE('knows').to(g.V('ben'))",
-            "g.V('ben').addE('knows').to(g.V('robin'))",
-            "g.V('thomas').property('age', 44)",
-            "g.V().count()",
-            "g.V().hasLabel('person').has('age', gt(40))",
-            "g.V().hasLabel('person').order().by('firstName', decr)",
-            "g.V('thomas').outE('knows').inV().hasLabel('person')",
-            "g.V('thomas').outE('knows').inV().hasLabel('person').outE('knows').inV().hasLabel('person')",
-            "g.V('thomas').repeat(out()).until(has('id', 'robin')).path()",
-            "g.V('thomas').outE('knows').where(inV().has('id', 'mary')).drop()",
-            "g.E().count()",
-            "g.V('thomas').drop()" };
+            // Submitting remote query to the server.
+            ResultSet results = client.submit(query);
 
-        for (String gremlin : gremlinQueries) {
-            ResultSet results = client.submit(gremlin);
+            CompletableFuture<List<Result>> completableFutureResults;
+            CompletableFuture<Map<String, Object>> completableFutureStatusAttributes;
+            List<Result> resultList;
+            Map<String, Object> statusAttributes;
 
-            CompletableFuture<List<Result>> completableFutureResults = results.all();
-            List<Result> resultList = completableFutureResults.get();
+            try{
+                completableFutureResults = results.all();
+                completableFutureStatusAttributes = results.statusAttributes();
+                resultList = completableFutureResults.get();
+                statusAttributes = completableFutureStatusAttributes.get();            
+            }catch(Exception e){
+                ResponseException re = (ResponseException) e.getCause();
+                System.out.println(Arrays.toString(re.getStatusAttributes().get().keySet().toArray()));
+                System.out.println("Status code: " + re.getStatusAttributes().get().get("x-ms-status-code")); 
+                System.out.println("Substatus code: " + re.getStatusAttributes().get().get("x-ms-substatus-code")); 
+                System.out.println("Request charge: " + re.getStatusAttributes().get().get("x-ms-request-charge"));
```

Comment:
I think that since we don't stream the results and use .all(), x-ms-request-charge isn't useful here, and we should just show x-ms-total-request-charge.